0000-settlement-engines/0000-settlement-engines.md

```diff
@@ -0,0 +1,395 @@
+---
+title: Settlement Engines
+draft: 1
+---
+
+# Settlement Engines
+
+## Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Overview
+
+This specification codifies a common interface for **settlement engines**. Settlement engines are services which perform two primary operations:
+
+1. Send outgoing settlements
+2. Receive incoming settlements
+
+Counterparties may operate compatible settlement engines to settle their liabilities between one another. Different implementations may utilize different settlement systems or types of settlements, such as:
```

Comment:
Different implementations may utilize different settlement systems or types of settlements, while some may not perform automated settlement at all.